docs/update_and_migration/migrate_to_ibexa_dxp/common_issues.md

```diff
@@ -38,6 +38,10 @@ In that case, run the `ezplatform:images:normalize-path` command to normalize th
 php bin/console ezplatform:images:normalize-path
 ```
 
+!!! note
+
+    If you have a corrupted path detected, check if you have special character in the `var_dir` configuration.
```

Comment:
I don't know if it's the place for a fix. Anyway, there is no official fix yet. I have a way to fix this but we should wait for a more mature solution and integrate it in another PR iteration.

```suggestion
    If you have a corrupted path detected, check if you have special character in the `var_dir` configuration.
    If you have a special char, to fix it, you must:
    - In the configuration, remove the special character from the `var_dir` value.
    - On the file system, rename the directory to remove the special character.
    - In the database, update `ezimagefile` to replace `ezimagefile.filepath` column to remove the special character.
```